model/event/commit_queue_event.go

```diff
@@ -69,6 +69,16 @@ func LogCommitQueueConcludeTest(patchID, status string) {
 	logCommitQueueEvent(patchID, CommitQueueConcludeTest, data)
 }
 
+func LogCommitQueueConcludeWithError(patchID, status string, err error) {
+	data := &CommitQueueEventData{
+		Status: status,
+	}
+	if err != nil {
+		data.Error = err.Error()
```

Comment:
Since we just convert this to a string anyway, can we just pass in a string? (can maybe call the function WithMessage instead)